translations/id-ID

```diff
@@ -0,0 +1 @@
+SelfName Indonesian
```

Comment:
Not "Bahasa Indonesia"?

Reply:
we people indonesia say Bahasa Indonesia, but word say Indonesian in list